### PR TITLE
Content tweaks for household situation section

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
* Adding ‘or’ dividers
* Order of options
* Question URL slugs

### CODE REVIEW

File: config/forms/2021_2022.json [hunk 1]
```diff
@@ -4365,10 +4365,10 @@
                       "value": "Internal transfer"
                     },
                     "2": {
-                      "value": "Tenant applied directly (no referral or nomination)"
+                      "value": "Tenant applied directly (no referral)"
                     },
                     "3": {
-                      "value": "Private registered provider (PRP) supported lettings only – referred by local authority housing department"
+                      "value": "Referred by local authority housing department"
```
Comment (paulrobertlloyd) [inline config/forms/2021_2022.json:4371]: @roobottom Do we need to include the text ‘Private registered provider (PRP) supported lettings only’ given the option is only shown in this case anyway?

File: config/forms/2021_2022.json [hunk 2]
```diff
@@ -4425,10 +4425,10 @@
                       "value": "Tenant applied directly (no referral or nomination)"
                     },
                     "3": {
-                      "value": "Private registered provider (PRP) lettings only – nominated by a local housing authority"
+                      "value": "Nominated by a local housing authority"
                     },
                     "4": {
-                      "value": "Private registered provider (PRP) supported lettings only – referred by local authority housing department"
+                      "value": "Referred by local authority housing department"
```
Comment (paulrobertlloyd) [inline config/forms/2021_2022.json:4431]: @roobottom Do we need to include the text ‘Private registered provider (PRP) [supported] lettings only’ given these options are only shown in this case anyway?

File: config/forms/2021_2022.json [hunk 3]
```diff
@@ -4308,10 +4308,10 @@
                       "value": "Internal transfer"
                     },
                     "2": {
-                      "value": "Tenant applied directly (no referral or nomination)"
+                      "value": "Tenant applied directly (no nomination)"
                     },
                     "3": {
-                      "value": "Private registered provider (PRP) lettings only – nominated by a local housing authority"
+                      "value": "Nominated by a local housing authority"
```
Comment (paulrobertlloyd) [inline config/forms/2021_2022.json:4314]: @roobottom Do we need to include the text ‘Private registered provider (PRP) lettings only’ given the option is only shown in this case anyway?

**Review by paulrobertlloyd — COMMENTED** [review]
@roobottom I have some questions about the labels we use for source of referral.

File: config/forms/2021_2022.json [hunk 4]
```diff
@@ -3611,7 +3611,7 @@
                 }
               ]
             },
-            "underoccupation_benefitcap": {
+            "under_occupation_benefit_cap": {
```
Comment (baarkerlounger) [inline config/forms/2021_2022.json:3614]: Why is this changing? Looks like the original was correct in the General Needs Data Dictionary?

Comment (paulrobertlloyd) [reply config/forms/2021_2022.json:3614]: This is just changing the URL slug, the data key stays the same.

**Pull request merged**